src/hotspot/cpu/riscv/nativeInst_riscv.cpp

```diff
@@ -265,6 +265,13 @@ void NativeJump::verify() { }
 
 
 void NativeJump::check_verified_entry_alignment(address entry, address verified_entry) {
+  // Patching to not_entrant can happen while activations of the method are
+  // in use. The patching in that instance must happen only when certain
+  // alignment restrictions are true. These guarantees check those
+  // conditions.
+
+  // Must be 4 byte aligned
+  MacroAssembler::assert_alignment(verified_entry);
```

Comment:
So, if the intent to pass explicit `NativeInsn::insn_size` everywhere we say "Must be 4 bytes aligned", then we should probably pass it here too. Or, drop the explicit `NativeInsn::insn_size` everywhere else?

Reply:
Sorry, I misread the patch, and confused `align` and `assert_alignment`.

Reply:
> Sorry, I misread the patch, and confused `align` and `assert_alignment`.

That's nothing. Thanks!

Reply:
I think this comment should be: "// Must be 4 bytes aligned"

Reply:
> I think this comment should be: "// Must be 4 bytes aligned"

Oh thanks for catching that. With hindsight, I found Aleksey had already hinted at it by indicating `...everywhere we say "Must be 4 bytes aligned"...`, but I failed to catch that :-) Thank you both!